src/BinSkim.Rules/RulesExtensionMethods.cs

```diff
@@ -153,15 +154,64 @@ private static string CreateLibraryKey(ObjectModuleDetails objectModuleDetail)
                 { CryptoError.CERT_E_UNTRUSTEDTESTROOT, "The certification path terminates with the test root which is not trusted with the current policy settings." },
                 { CryptoError.CERT_E_VALIDITYPERIODNESTING, "The validity periods of the certification chain do not nest correctly." },
                 { CryptoError.CERT_E_WRONG_USAGE, "The certificate is not valid for the requested usage." },
+                { CryptoError.CERTSRV_E_ADMIN_DENIED_REQUEST, "The request was denied by a certificate manager or CA administrator." },
+                { CryptoError.CERTSRV_E_ALIGNMENT_FAULT, "A memory reference caused a data alignment fault." },
+                { CryptoError.CERTSRV_E_ARCHIVED_KEY_REQUIRED, "The request is missing a required private key for archival by the server." },
+                { CryptoError.CERTSRV_E_ARCHIVED_KEY_UNEXPECTED, "The request includes a private key for archival by the server, but key archival is not enabled for the specified certificate template." },
```

Comment:
"for archiv**ing** by the server" scans a little better for me, but none of my public school English teachers were as excited about teaching grammar as discussing literature, so I couldn't explain exactly why. I might be in error. :)

Reply:
Fixed, the text was from Windows SDK, both sounds good to me.

Reply:
Oh man, if I'd known you were using strings that were already public-facing I wouldn't have said anything.  Thanks though!